### DISCUSSION TITLE
Using wfb-ng for video, control signal and telemetry

### DISCUSSION THREAD

**Discussion by parijip** [svpcom/wfb-ng#261] [General]
Hi,
I am a complete novice to wfb-ng and from my initial read of documents I gather, it will be possible to transmit video, mavlink telemetry as well as control signals between drone and ground control just using wfb-ng.
Is that a fair assumption. And in that case is it fair to assume that I can avoid using a separate radio transmitter/receiver combo when using wfb-ng.
Also, if I enable encryption, what is the likely cost that I am to incur in terms of latency - if I run wfb-ng air component on a jetson nano on the bird? Is there any other cost associated with enabling encryption on wfb-ng?
Thanks in advance.

**Comment by svpcom** [comment]
Yes, you don't need rc-radio transmitter. Encryption will not add any additional costs

**Reply by parijip** [reply]
I have 1 more question:
Are the channels for video, telemetry and control different? Is it possible to have the control and telemetry hopping across the available channels in the band? Is it possible to hop across 2.4 and 5.8 GHz channels? Is it possible to transmit on both bands to ensure higher reliability?

**Reply by svpcom** [reply]
No. Current code doesn't support channel hopping during run.  wifi card unable to transmit on different freq channels simultaneously. You could setup several wfb-ng instances using different cards to use several channels in parallel
